src/wp-includes/pluggable.php

```diff
@@ -2384,7 +2384,15 @@ function wp_salt( $scheme = 'auth' ) {
 
 		static $duplicated_keys;
 		if ( null === $duplicated_keys ) {
-			$duplicated_keys = array( 'put your unique phrase here' => true );
+			$duplicated_keys = array(
+				'put your unique phrase here'       => true,
+				/*
+				* translators: This string should only be translated if wp-config-sample.php is localized.
+				* You can check the localized release package or
+				* https://i18n.svn.wordpress.org/<locale code>/branches/<wp version>/dist/wp-config-sample.php
+				*/
```

Comment:
```suggestion
				/*
				 * translators: This string should only be translated if wp-config-sample.php is localized.
				 * You can check the localized release package or
				 * https://i18n.svn.wordpress.org/<locale code>/branches/<wp version>/dist/wp-config-sample.php
				 */
```

A minor alignment nitpick.

The comment matches the equivalent used elsewhere so that's perfect, thank you!